payjoin/tests/integration.rs

```diff
@@ -74,7 +74,9 @@ mod integration {
         struct HeaderMock(HashMap<String, String>);
 
         impl Headers for HeaderMock {
-            fn get_header(&self, key: &str) -> Option<&str> { self.0.get(key).map(|e| e.as_str()) }
+            fn get_header(&self, key: &str) -> Option<&str> {
+                self.0.get(key).map(|e| e.as_str())
+            }
```

Comment:
Using a different formatter than nightly? Or was nightly updated?

Reply:
Ah, I was on stable. Thoughts on adding a [rust-toolchain.toml](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file) to the project?